views/sign-out.html

```diff
@@ -27,7 +27,9 @@
         <div class="reload-message">
           <h1>{{ gettext("pleaseSignOut") }}</h1>
           <p>{{ gettext("pleaseSignOutMessage") }}</p>
-          <a href="{{ logoutURL }}" class="action-button" target="_self">{{ gettext("signOutButtonLabel") }}</a>
+          <p class="button-wrapper">
```

Comment:
nit: `<p>` vs. `<div>`?

Reply:
Have kept as ``p`` since the other page that uses the same markup has a ``p`` so will leave for now.